libs/community/langchain_google_community/vertex_ai_search.py

```diff
@@ -279,23 +278,6 @@ class VertexAISearchRetriever(BaseRetriever, _BaseVertexAISearchRetriever):
     https://cloud.google.com/generative-ai-app-builder/docs/boost-search-results
     https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1beta/BoostSpec
     """
-    custom_embedding: Optional[Embeddings] = None
```

Comment:
Hi @lkuligin , 

The main reason for this change is stability. When using v1beta, the retrieval times are inconsistent, sometimes taking up to 10x longer than v1. For example, there were cases where fetching results took 20 seconds instead of the expected 0.5 seconds. This kind of delay is not acceptable for a tool designed to support an agent efficiently.

Do you think we should maintain compatibility by supporting both the beta and stable versions in the class, or is it better to fully transition to the stable version?

Reply:
@lkuligin let me know your thoughts, and we can align on the best approach moving forward.

Reply:
I'd suggest to maintain both clients, and if beta arguments are set, throw a warning and use v1beta client in that case (the default one should be v1).
Thanks!